README.md

```diff
@@ -108,6 +108,27 @@ async function initialize() {
 initialize();
 ```
 
+## Creating a typed Content Editor extensions with Typescript
```

Comment:
Mixing plural and singular (realise it's following the same pattern of the previous types but they should all be changed)
> Creating a typed Content Editor extension**s** with Typescript

to

> Creating a typed Content Editor extension with Typescript